src/autoimport/services.py

```diff
@@ -4,12 +4,58 @@
 and handlers to achieve the program's purpose.
 """
 
-from typing import Any, Dict, Optional, Tuple
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple
 
 from _io import TextIOWrapper
+from maison.config import ProjectConfig
 
 from autoimport.model import SourceCode
 
+# To consider: use `xdg.xdg_config_home() / "autoimport" / "config.toml"`
```

Comment:
I feel that the merging of multiple files could be supported at `maison` level. Let me ask the developer if he'd like the feature, so we can save all this logic

Reply:
You can follow up in [this issue](https://github.com/dbatten5/maison/issues/61)

Reply:
@dbatten5 agreed on having the logic at `maison` level, do you mind migrating the code to a PR in their repo?

Reply:
Hey @HoverHell , the feature is available on the new version of maison (`1.3.0`)!